backend/coins/eth/coin.go

```diff
@@ -120,8 +120,8 @@ func (coin *Coin) Unit(isFee bool) string {
 }
 
 // GetFormatUnit implements coin.Coin.
-func (coin *Coin) GetFormatUnit() string {
-	return coin.unit
+func (coin *Coin) GetFormatUnit(isFee bool) string {
```

Comment:
add unit tests for Unit() and FormatUnit() for ETH/ERC20 while you are here? :smile:

Reply:
Done :+1: